tests/Oryx.BuildImage.Tests/golang/GolangDynamicInstallationTest.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace Microsoft.Oryx.BuildImage.Tests
 {
-    [Trait("platform", "golang")]
+    [Trait("platform", "golang-dynamic")]
```

Comment:
Is this golang dynamic installation test used for go `full` image, should we name it as `golang-full` or other names?

Reply:
Sounds good to me, I've made the change. Thanks!